src/bsdd/__about__.py

```diff
@@ -0,0 +1,4 @@
+# SPDX-FileCopyrightText: 2024-present jgunstone <j.gunstone@maxfordham.com>
+#
+# SPDX-License-Identifier: MIT
```

Comment:
bSDD is under LGPL-3.0-or-later - would it be possible to change this?

Also the version is not tied to IfcOpenShell, so I guess we can start at whatever number we want (probably not 1.0.0 because it's probably not "stable" as yet, still experimental and bSDD is changing continuously, so maybe 0.1.0 or 0.0.1?)

Reply:
yikes! apologies, didn't mean to change the license! will update now